dal/database.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 const mongoose = require('mongoose');
 const Product = require('../schemas/product');
-const uri = process.env.MONGODB_URI || '127.0.0.1';
+const uri = process.env.MONGODB_URI || 'mongodb://localhost:27017/test' || '127.0.0.1';
```

Comment:
We should only have two options here. Lets change `process.env.MONGODB_URI || 'mongodb://localhost:27017/test' || '127.0.0.1';` to `process.env.MONGODB_URI || 'mongodb://localhost:27017/test';`.